### PR TITLE
Feature/front end

### PR DESCRIPTION
**Pull request by ajthomascouk**

### CODE REVIEW

**Comment by st3v3nhunt** [comment]
@ajthomascouk Some conflicts need resolving please

File: Dockerfile [hunk 1]
```diff
@@ -15,4 +15,6 @@ EXPOSE 3000
 
 COPY . /code
 
+RUN [ "npm", "run" ]
```
Comment (st3v3nhunt) [inline Dockerfile:18]: Does this need to be here?

File: app/views/includes/foot.nunjucks [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<script src="{{ SITE_ROOT }}/js/jquery-1.12.4.min.js" type="text/javascript"></script>
+<script src="{{ SITE_ROOT }}/js/cookie-message.js" type="text/javascript"></script>
+<script src="{{ SITE_ROOT }}/js/selection-buttons.js" type="text/javascript"></script>
+<script src="{{ SITE_ROOT }}/js/geolocation.js" type="text/javascript"></script>
```
Comment (st3v3nhunt) [inline app/views/includes/foot.nunjucks:4]: Where is this used i.e. `geolocation.js`

File: app/views/search.nunjucks [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{% extends 'layout.nunjucks' %}
```
Comment (st3v3nhunt) [inline app/views/search.nunjucks:1]: Is this file required?

File: config/routes.js [hunk 4]
```diff
@@ -5,10 +5,6 @@ const setLocals = require('../app/middleware/setLocals');
 const log = require('../app/middleware/logger');
 const getGp = require('../app/middleware/getGp');
 
-router.get('/', (req, res) => {
```
Comment (st3v3nhunt) [inline config/routes.js:8]: Why has this gone?

File: docker-compose.yml [hunk 5]
```diff
@@ -30,4 +30,3 @@ services:
 
 volumes:
   profiles-node_modules:
-
```
Comment (st3v3nhunt) [inline docker-compose.yml:33]: Leave this empty line in

File: public/js/geolocation.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
+jQuery(function($) {
```
Comment (st3v3nhunt) [inline public/js/geolocation.js:1]: Is this file required?

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]
Conflicts to resolve.

Does the geolocation stuff need to be included?

There are a bunch of image and js files in the `app/views` dir. Should they be in `/public` only?

It looks very much like a copy and paste. There are ways to share the code between repos - have these been considered? And what is the rationale for taking that approach?

A bunch of tests are failing on the build. Mostly because the test is along the lines of `expect(res.text).to.equal(${some-word});`. They should work ok if they are changed to `expect(res.text).to.contain(${some-word});`

**Comment by st3v3nhunt** [comment]
[style front-end for GP profile pages](https://trello.com/c/RpZoL5iS/840-style-front-end-for-gp-profile-pages)

File: Dockerfile (hunk 1 above)
Comment (ajthomascouk) [reply Dockerfile:18]: @st3v3nhunt - Im not sure on this one. Could do with your input on this.

File: app/views/includes/foot.nunjucks (hunk 2 above)
Comment (ajthomascouk) [reply app/views/includes/foot.nunjucks:4]: Removed this. It was left over from a copy and paste dump.

File: app/views/search.nunjucks (hunk 3 above)
Comment (ajthomascouk) [reply app/views/search.nunjucks:1]: @st3v3nhunt - I guess its not if this app is only one page?

File: config/routes.js (hunk 4 above)
Comment (ajthomascouk) [reply config/routes.js:8]: I put it back in. I wasn't sure that it was actually needed/useful.

File: public/js/geolocation.js (hunk 6 above)
Comment (ajthomascouk) [reply public/js/geolocation.js:1]: Removed

**Comment by st3v3nhunt** [comment]
@ajthomascouk Is this ok to close now?

**Comment by ajthomascouk** [comment]
No longer needed.

**Pull request closed without merging**